tsdb/cursors/schema.go

```diff
@@ -0,0 +1,37 @@
+package cursors
+
+// FieldType represents the primitive field data types available in tsm.
+type FieldType int
+
+const (
```

Comment:
Alternatively I think you can shorten this up to:

```go
const (
	Float    FieldType = iota // means the data type is a float
	Integer                   // means the data type is an integer
	Unsigned                  // means the data type is an unsigned integer
	Boolean                   // means the data type is a boolean
	String                    // means the data type is a string of text
)
```

Reply:
Yes, you are right. I was originally going to use `iota` and then decided I didn't want the constants to change, but they are never serialized, so it does not matter.